membership-attribute-service/conf/routes

```diff
@@ -2,6 +2,8 @@ GET         /healthcheck
 
 GET         /user-attributes/me                                                 controllers.AttributeController.attributes
 
+GET         /user-attributes/me/existing-payment-options                        controllers.ExistingPaymentOptionsController.existingPaymentOptions(cardEnabled: Boolean, directDebitEnabled: Boolean, currencyFilter: String)
```

Comment:
I'm not quite sure I understand the benefit of having a client pass through card/direct debit enabled - would you mind explaining the rationale for this?

Reply:
The main initial client `support-workers` will be sending these flags (see https://github.com/guardian/support-admin-console/pull/14). In the case where the login is valid but older than an hour, currently the response from this new endpoint will be really lightweight essentially just revealing how many (not any details about the type of payment method) meaning it can't be filtered client-side. 
If this is a problem, I could introduce an additional field (present regardless of staleness of the login) which describes the payment type, so it could filter client-side  - however IMO this is just the same net complexity. This approach also has the minor benefit of smaller response payload in the event that one of the switches is turned OFF.

Reply:
I'm not too keen on members-data-api having to care about switch state which belongs to `support-frontend` - IMO the responsibility for filtering is there. Not a huge problem though so if it's a lot of work feel free to disregard this comment

Reply:
OK will move it to the client

Reply:
done in https://github.com/guardian/members-data-api/pull/376/commits/0d360cb9b9e671f3e2b501b29e57d1368072779a (plus accompanying UI change https://github.com/guardian/support-frontend/pull/1673/commits/e9fc837b18e70edb819811e9f098a7e29f4c3aad)